dhis-2/dhis-api/src/main/java/org/hisp/dhis/keyjsonvalue/KeyJsonValueService.java

```diff
@@ -125,6 +125,8 @@
      */
     void deleteKeyJsonValue( KeyJsonValue entry );
```

Comment:
General comment: Add javadoc to all interface methods and all private methods.

Reply:
fixed